account/models.py

```diff
@@ -1,3 +1,56 @@
 from django.db import models
 
-# Create your models here.
+from pin.models     import (
+    Board,
+    Pin,
+    Topic,
+    PinUser,
+    PinTopic,
+    Comment,
+    UserTopic
+)
+
+class User(models.Model):
+    username        = models.CharField(max_length=45, null=True)
```

Comment:
max_length 45보다는 50으로 하는게 낫지않을까요
```suggestion
    username        = models.CharField(max_length=50, null=True)
```